documentation/how-to/gnss.en.md

```diff
@@ -7,8 +7,8 @@ tx_slug: documentation_how-to_gnss
 
 QField can make use of the internal GNSS (Global Navigation Satellite
 System, like GPS, GLONASS, Galileo or Beidou). QField can also connect
-to external antennas through bluetooth and directly integrate their NMEA
-stream.
+to external antennas through NMEA streams over Bluetooth, TCP, or UDP
+connection.
```

Comment:
Shall we add a compatibility table OS/technology?

Reply:
Android: Bluetooth, TCP, UDP
iOS: TCP, UDP
Windows: TCP, UDP
Linux: Bluetooth, TCP, UDP

@m-kuhn , correct?

Reply:
@m-kuhn , done.